Header header
geometry_msgs/Point[] goal_points
